reconcile/gitlab_labeler.py

```diff
@@ -30,6 +86,12 @@ def guess_labels(project_labels, changed_paths):
                        and t not in ignore_tokens]
             if matches:
                 guesses.add(label)
```

Comment:
We discussed offline yesterday but writing it here for the benefit of the rest of the reviewers. I think you have to remove this old logic of finding apps. If not, we could end with many `tenant-insights` MRs with different onboarding-status, which is not what we want.

For those apps with parent app, I would include both labels, e.g. `tenant-insights` `tenant-rbac` so that it is clearer (especially since there are so many insights apps).